src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -470,6 +471,12 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
         },
         ov::pass::NormalizeL2Decomposition);
 
+    CPU_SET_CALLBACK_X64(manager,
+        [](const_node_ptr &node) -> bool {
+            return !node->is_dynamic() && node->get_element_type() == element::f32;
```

Comment:
If `inference_precision=bf16`, this GN will be tokenized by Snippets. 
If the PR supports only f32, we should add check here that `inference_precision=f32` to allow Snippets to tokenize GN.
What do you think?

Reply:
good catch, applied, thanks Alexandra!

Reply:
Just one moment 😃 
I'd prefer check for `inference_precision == f32` as I noticed above since some CPU x64 [support](https://github.com/openvinotoolkit/openvino/blob/9e759e4758ad87d019432b077bbb5bac58a39233/src/plugins/intel_cpu/src/config.cpp#L274) fp16 precision.
Thus, the check `inference_precision == f32` is safer than `inference_precision != bf16`. What do you think?

Reply:
@a-sidorova , agree and f32 is used. two conditions are also updated as discussed offline. i.e. tensor_size and work_amount. Thanks Alexandra!